lib/linguist/languages.yml

```diff
@@ -4467,6 +4467,7 @@ SSH Config:
   extensions:
   - ".ssh_config"
   - ".ssh-config"
+  - ".sshconfig"
```

Comment:
Crap, I just realised we're missing the `ssh_config` filename (in addition to the usual extensions). There are [26,940 cases](https://github.com/search?q=filename%3Assh_config+NOT+nothack&type=Code) of this filename used on GitHub, so this needs to be catered for too.

Reply:
Then also `sshconfig` to filenames ? [5,377 results](https://github.com/search?q=filename%3Asshconfig+NOT+nothack&type=Code)

Reply:
Ah yes, good catch! Had no idea there were so many filenames in use... 😅

Reply:
We'd better add the hyphenated variant too… [26,940 results](https://github.com/search?q=filename%3Assh-config+NOT+nothack&type=Code). 😕

Reply:
sshd_config has [24,473 code results
](https://github.com/search?q=filename%3Asshd_config+NOT+nothack&type=Code)
sshd_config has also the same. Wonder why...

So basically every extension is also a filename :)